modules/howtos/pages/slow-operations-logging.adoc

```diff
@@ -1,7 +1,7 @@
 = Slow Operations Logging
 :description: Tracing information on slow operations can be found in the logs as threshold logging, orphan logging, and other span metrics.
 :page-topic-type: howto
-// :page-aliases: ROOT:
+:page-aliases: ROOT:slow-operations-logging.adoc
```

Comment:
not needed (no 2.7 page with this name),
but on't forget to check the two 2.7 pages are aliased in only one place each.

Reply:
Yeah, I'd like to understand the 2.7 part a bit better and `page-aliases` in general 😓